frontend/src/store/index.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * Redux Store with Redux Toolkit & RTK Query
+ *
+ * @see https://redux-toolkit.js.org/tutorials/typescript
+ * @see https://redux-toolkit.js.org/tutorials/rtk-query#add-the-service-to-your-store
+ */
+
+// TODO We shouldn't have 2 stores, it's better to persist each root slice reducer. This is a bad pattern:
+// https://redux.js.org/faq/store-setup#can-or-should-i-create-multiple-stores-can-i-import-my-store-directly-and-use-it-in-components-myself
+
+import { combineReducers, configureStore, isPlain } from '@reduxjs/toolkit'
+import { setupListeners } from '@reduxjs/toolkit/query'
+import { createTransform, FLUSH, PAUSE, PERSIST, PURGE, REGISTER, REHYDRATE } from 'redux-persist'
+import persistReducer from 'redux-persist/es/persistReducer'
+import persistStore from 'redux-persist/es/persistStore'
+import autoMergeLevel2 from 'redux-persist/es/stateReconciler/autoMergeLevel2'
+import storage from 'redux-persist/es/storage' // LocalStorage
+
+import { missionApi } from '../api/mission'
+import { mapToProcessingRegulation } from '../domain/entities/regulation'
+import { persistedRootReducer, rootReducer } from '../domain/shared_slices'
+
+import type { AnyAction } from '@reduxjs/toolkit'
+import type { PersistConfig } from 'redux-persist'
+import type { ThunkAction } from 'redux-thunk'
+
+// =============================================================================
+// Common Store
+
+// TODO Properly type this generic. No any.
+const SetTransform = createTransform(
```

Comment:
This is only used by the `persistedReducerConfig`, it might be moved to this section